Project-Hotpot/Project-Hotpot/ViewControllers/Playlist/PlaylistDetailsViewController.swift

```diff
@@ -70,4 +70,34 @@ class PlaylistDetailsViewController: MediaViewController, UITableViewDelegate, U
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return songArray?.count ?? 0
     }
+    
+    func removeSongFromPlaylist(song: PFSong) {
```

Comment:
Maybe better suited to live in SongManager and just call from here. Reduces Controller lines of code and keeps it clean. Ideally we have ViewControllers --Request-->Manager--Manipulate--->DB Objects